common/constants.js

```diff
@@ -1,4 +1,4 @@
-const API_EVENTS_URL = '/api/v1/splash-events/?start_time__gte=2018-08-01';
+const API_EVENTS_URL = '/api/v1/splash-events/?start_time__gte=' + new Date().getFullYear() + '-08-01';
```

Comment:
Having it like this is probably going to be irritating for velkom in the new year, when they are likely to want to use the splash to see the previous year's events?